### PR TITLE
test(ext/node): add _util_callbackify_test

### PR DESCRIPTION
**Pull request by weartist**

### CODE REVIEW

File: cli/tests/unit_node/_util/_util_callbackify_test.ts [hunk 1]
```diff
@@ -0,0 +1,370 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { assert, assertStrictEquals } from "../../../../test_util/std/testing/asserts.ts";z
+import { callbackify } from "node:_util/_util_callbackify";
```
Comment (kt3k) [inline cli/tests/unit_node/_util/_util_callbackify_test.ts:4]: callbackify is available from `"node:util"`

**Comment by kt3k** [comment]
Run `./tools/format.js` to format the code.

You can run this test locally with the command `cargo test callbackify_test`

File: cli/tests/unit_node/_util/_util_callbackify_test.ts (hunk 1 above)
Comment (weartist) [reply cli/tests/unit_node/_util/_util_callbackify_test.ts:4]: oh，thanks！

**Comment by weartist** [comment]
> 

thanks，haha

**Review by aapoalas — CHANGES_REQUESTED** [review]
Sorry, can you rebase the changes? Thank you <3

**Pull request closed without merging**